### PR TITLE
Fix Style/MultilineTernaryOperator offenses

### PR DESCRIPTION
**Pull request by magni-**
Yet another small PR from #752

### CODE REVIEW

**Comment by jaredbeck** [comment]
These are all fine. I'd prefer to extract private methods, but that can happen later. 

Please rebase.

File: lib/paper_trail/reifier.rb [hunk 1]
```diff
@@ -284,9 +282,8 @@ def reify_has_many_through(transaction_id, associations, model, options = {})
       # table, this method would return the constant `Fruit`.
       def version_reification_class(version, attrs)
         inheritance_column_name = version.item_type.constantize.inheritance_column
-        class_name = attrs[inheritance_column_name].blank? ?
```
Comment (magni-) [inline lib/paper_trail/reifier.rb]: Do people ever set `inheritance_column` to `""` ? That's the only case in which this would break...

Happy to revert this change if needed. Looking through `git`, it's been `blank?` since the code was first written in 2009 (so not in response to a bug report from someone using `inheritance_column = ""`).

Comment (jaredbeck) [inline lib/paper_trail/reifier.rb]: > Do people ever set inheritance_column to "" ?

Seems unlikely, but we should be changing behavior in a linter commit. If we are going to make that change, please make a separate PR. Thanks.

File: lib/paper_trail/has_paper_trail.rb [hunk 2]
```diff
@@ -306,6 +303,14 @@ def source_version
         send self.class.version_association_name
       end
 
+      def preceding_version
+        if source_version
+          source_version.previous
+        else
+          send(self.class.versions_association_name).last
+        end
+      end
```
Comment (jaredbeck) [inline lib/paper_trail/has_paper_trail.rb]: I know I said I'd prefer to extract private methods, but we can't do that here because we already have a problem with polluting people's AR models with a ton of methods, and I don't want to make that problem worse. See discussion in https://github.com/airblade/paper_trail/pull/719

**Comment by jaredbeck** [comment]
Closed by https://github.com/airblade/paper_trail/pull/768
